tests/integration_tests/data_steward/gcs_utils_test.py

```diff
@@ -11,27 +11,29 @@
 import bq_utils
 import gcs_utils
 from gcloud.gcs import StorageClient
+from gcloud.bq import BigQueryClient
 from tests import test_util
 from tests.test_util import FIVE_PERSONS_PERSON_CSV, FAKE_HPO_ID
 
 
 class GcsUtilsTest(unittest.TestCase):
 
     dataset_id = bq_utils.get_dataset_id()
+    project_id = app_identity.get_application_id()
+    bq_client = BigQueryClient(project_id)
```

Comment:
I think `bq_client = BigQueryClient(project_id)` can be placed here or within `setUpClass(cls)`. Is there any particular reason you chose here over `setUpClass(cls)`? (I am just curious to know :slightly_smiling_face:)

Reply:
I remember I was asked to place a `StorageClient` in the same place when working on PR updates a while back. So, I decided to place it there to stay consistent with `dataset_id` and the `StorageClient`. If you think it would be better to place it in `setUpClass(cls)` then I can I do that.🙂